server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+
+const app = express();
+const port = 3000;
+
+app.use(express.static('dist/ys-pet'));
+
+const apiRouter = require('./src/api/api');
+
+app.use('/api', apiRouter);
+
+app.get('/', (req, res) => {
+  res.sendFile(__dirname + '/dist/ys-pet/index.html');
+});
+
+app.get('*', (req, res) => {
```

Comment:
This one will send 404 for user accessing angular route first (ie http://mysite.com/feed, considering /feed - angular route for FeedPageComponent). Better send index.html for angular init that will either render component of 404